src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < 3; i++) {
+            figures[i] = new FigureSupplier().getRandomFigure();
```

Comment:
Ensure that `FigureSupplier.getRandomFigure()` does not return a direct instance of `Figure` if `Figure` is intended to be a base class or abstract. This aligns with the suggestion to make `Figure` an abstract class.